internal/provider/resource_alert_urgency_test.go

```diff
@@ -20,6 +20,9 @@ import (
 func init() {
 	resource.AddTestSweepers("rootly_alert_urgency", &resource.Sweeper{
 		Name: "rootly_alert_urgency",
+		Dependencies: []string{
+			"rootly_alerts_source",
+		},
```

Comment:
Is this in scope of this PR? If not, please remove.

Reply:
Will do! For context, most Rootly resources enforce a unique name constraint, so I've started to use randomly generated names prefixed with "tf-" so that tests can run concurrently without conflicts. These are test sweepers (`make sweeper`) that we can run and delete them.